tests/mkldnn_compat/examples/cpu_memory_format_propagation.cpp

```diff
@@ -102,10 +102,10 @@ void cpu_memory_format_propagation_tutorial() {
     ///
     /// To specify that a primitive should pick an optimized format for the
     /// specified computation parameters, we create memory descriptors with
-    /// memory format set to @ref mkldnn::memory::format_tag::any.
+    /// memory format set to @ref dnnl::memory::format_tag::any.
```

Comment:
I still suggest to not touch the `tests/mkldnn_compat/*` directory, as it is a direct copy-paste from the older MKL-DNN v1.0. Given that these files should not be processed by `doxygen`, the _incorrect_ (actually just outdated) links should not be a big issue.

Reply:
Agree